index.js

```diff
@@ -442,11 +457,12 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
         response.status(200).json(errorMessage)
       } else {
         const message = JSON.parse(request.body.data)
-        const doorStatus = message.door_status
-        const doorTime = message.door_time
-        const insTime = message.ins_time
+        const missedDoorMessages = message.md
+        const doorLowBattery = message.lb
+        const doorHeartbeatReceived = message.dh
```

Comment:
Just curious... is `doorHeartbeatReceived` going to be a timestamp? If so, which time zone will it be in?

Reply:
It will be a timestamp, in milliseconds relative to the last door heartbeat recorded

Reply:
That's a good way to get around the timezone issue :smile: